types/test.ts

```diff
@@ -0,0 +1,7 @@
+import stripe from 'tipsi-stripe'
+
+stripe.setOptions({
+  publishableKey: 'PUBLISHABLE_KEY',
+  merchantId: 'MERCHANT_ID', // Optional
+  androidPayMode: 'test', // Android only
+});
```

Comment:
this should be full of example usage, which is the "test" part of it, I took this example from the docs